scripts/docker-init.sh

```
@@ -1,12 +1,32 @@
 #!/usr/bin/env sh
-set -eu
+set -euo pipefail
```

Shellcheck (0.10.0)

[warning] In POSIX sh, set option pipefail is undefined.

(SC3040)